src/Microsoft.DotNet.Wpf/src/WindowsBase/MS/Internal/Permissions/CompoundFileIOPermission.cs

```diff
@@ -29,25 +11,16 @@
 
 namespace MS.Internal.Permissions
 {
-    // !!!! Warning !!!!: No code other than XPSViewer (or Mongoose) should assert this
-    //  permission without agreement from this code owners.
-    [Serializable]
     [FriendAccessAllowed]
     internal class CompoundFileIOPermission : InternalParameterlessPermissionBase
```

Comment:
This type can be deleted easily. It is only used in `SecurityHelper.DemandCompoundfileIOPermission`, which can be made into a no-op. 

In fact, `SecurityHelper.DemandCompoundFileIOPermission()` can be deleted as well. It has ~66 callers, each of which can likely be safely deleted (it's signature is `void(void)`, so erasing it has no side effects).